<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QWidget" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="StyledButton" name="btnMsg" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnHome" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="StyledButton" name="btnMine" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledButton</class>
   <extends>QWidget</extends>
   <header>widgets/StyledButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
